### PR TITLE
Style

### PR DESCRIPTION
**Pull request by muhenge**
# Work done

- Added style to the project using bootstrap framework

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that not all the linter checks are configured. In your `github/workflows/linters.yml` file, you only have configurations for eslint (JS) and stylelint (CSS). You need to complete the other configurations, since this is an HTML/CSS/JS project. See config [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js)

Do not forget to delete the `.vscode` directory from your commit history, and add it to your `.gitignore` file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by muhenge** [comment]
> Hi team,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that not all the linter checks are configured. In your `github/workflows/linters.yml` file, you only have configurations for eslint (JS) and stylelint (CSS). You need to complete the other configurations, since this is an HTML/CSS/JS project. See config [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js)
> 
> Do not forget to delete the `.vscode` directory from your commit history, and add it to your `.gitignore` file.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 
> 
> Cheers and Happy coding!
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by muhenge** [comment]
> > Hi team,
> > While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> > ## Invalid Code Review Request
> > You have submitted a project, that not all the linter checks are configured. In your `github/workflows/linters.yml` file, you only have configurations for eslint (JS) and stylelint (CSS). You need to complete the other configurations, since this is an HTML/CSS/JS project. See config [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js)
> > Do not forget to delete the `.vscode` directory from your commit history, and add it to your `.gitignore` file.
> > Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready
> > Cheers and Happy coding!
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by muhenge** [comment]
Hello @teekaytech, We early add the .vscode file in the .gitignore file, if you verify again you will find it

**Comment by teekaytech** [comment]
> Hello @teekaytech, We early add the .vscode file in the .gitignore file, if you verify again you will find it

Hi,

You probably did, but it looks like you added it to your gitignore after you've already committed the file. That is why you still have the folder in your project despite the fact that it's added.

You may delete the folder, then commit your changes. That should do.

**Comment by muhenge** [comment]
@teekaytech Well noted

**Review by smcommits — COMMENTED** [review]
Hi @muhenge, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Nice work adding the linters as requested by the previous reviewer. However, your repo is still missing HTML linters. Without linters, the code review request will be invalid as linters take care of a lot of common errors. Please refer to the guide provided by the previous reviewer. Make sure to add linters for HTML to your Github actions. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by muhenge** [comment]
Hello @smcommits , Thanks for the review, following your request, I would like to ask you to please specify what exactly missing in the project, because we added the `hint` dev dependency for HTML lint, we followed the guidance [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js), it is not clear for us, thank you.

**Comment by smcommits** [comment]
> Hello @smcommits , Thanks for the review, following your request, I would like to ask you to please specify what exactly missing in the project, because we added the `hint` dev dependency for HTML lint, we followed the guidance [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js), it is not clear for us, thank you.

@muhenge Hi, hope you are doing great. So, the guidelines you followed about adding `hint` to dev dependency is to set up the linters on the local machine. When you create a pull request, you might notice the following checks initialized: 

image

We call this practice [CI, CD](https://docs.github.com/en/actions/guides/about-continuous-integration), and we use Github actions to achieve this. What this does is that it allows running linters on the repo itself, independent of any machine. When you copy .github/worflows folder, you set up what I just described. 

Coming to your repo, as you can see in the image above, the checks for HTML linters are not initialized on your repo. This might be because you did not copy the correct workflows folder. Please copy the .github/workflows folder in [this repo](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to your repo. 

Please feel free to leave comments if you have any questions. I hope I was of any help.

**Comment by muhenge** [comment]
> > Hello @smcommits , Thanks for the review, following your request, I would like to ask you to please specify what exactly missing in the project, because we added the `hint` dev dependency for HTML lint, we followed the guidance [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js), it is not clear for us, thank you.
> 
> @muhenge Hi, hope you are doing great. So, the guidelines you followed about adding `hint` to dev dependency is to set up the linters on the local machine. When you create a pull request, you might notice the following checks initialized:
> 
> image
> 
> We call this practice [CI, CD](https://docs.github.com/en/actions/guides/about-continuous-integration), and we use Github actions to achieve this. What this does is that it allows running linters on the repo itself, independent of any machine. When you copy .github/worflows folder, you set up what I just described.
> 
> Coming to your repo, as you can see in the image above, the checks for HTML linters are not initialized on your repo. This might be because you did not copy the correct workflows folder. Please copy the .github/workflows folder in [this repo](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to your repo.
> 
> Please feel free to leave comments if you have any questions. I hope I was of any help.

Noted

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @muhenge** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- Your project looks good, but you need to fix the title and author issue [ see screenshot here](https://user-images.githubusercontent.com/61562626/125689976-65374a35-5274-4f6e-9f13-99b20aa0e57d.png)👈👈

- I can't tell the title from the author.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -1,66 +1,87 @@
...
+  
+      for (const book of documentVariables.booksCatalog) {
+        const bookCont = document.createElement('div');
+        const titleAuth = document.createElement('div');
+        const titleAuthClasses = 'd-flex justify-content-around'.split(' ');
+        titleAuth.classList.add(...titleAuthClasses);
+        const bookName = document.createElement('span');
+        bookName.classList.add('d-inline');
+        const authorName = document.createElement('span');
+        authorName.classList.add('d-inline');
```
Comment (dasileker) [inline index.js]: - [x] Please add headers to the span so the title has a title and the author have one also.

Comment (muhenge) [reply index.js]: Well noted

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @muhenge and @webdevmaxi , 

Your project is complete! you did an amazing job guys.. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**